### PR TITLE
Fix full example signup error caused by no name provided

### PR DESCRIPTION
**Pull request by oiojin831**

### CODE REVIEW

File: examples/full-example/src/email-password/signup.ts [hunk 1]
```diff
@@ -82,6 +82,7 @@ async function createGraphcoolUser(api: GraphQLClient, email: string, password:
   const variables = {
     email,
     password: password,
+    name: name,
```
Comment (kbrandwijk) [inline examples/full-example/src/email-password/signup.ts]: Only this one got left behind.

Comment (oiojin831) [reply examples/full-example/src/email-password/signup.ts]: @kbrandwijk sorry :(
I just changed it.

**Comment by marktani** [comment]
@oiojin831 can you elaborate what the problem is you're trying to fix? would @kbrandwijk's suggestion alleviate this problem?

**Comment by kbrandwijk** [comment]
I think it just comes down to making `name` optional (`String!` -> `String`) in this full-example.

**Comment by oiojin831** [comment]
@marktani 
@kbrandwijk is pointing out correctly.
Should I just change ```name``` to be optional?

**Comment by nikolasburk** [comment]
@oiojin831 I think making `name` optional is the proper way here - I'll merge the PR once that's done. Thanks!

**Pull request closed without merging**